### PR TITLE
Add info about time sync

### LINKED ISSUE #778: Add blurb on NTP Time Sync for hosts
**Summary**

Operations in any distributed system tend to require accurate system clocks.

Most hosts should have a built-in way to synchronize time, and we should add that to our software checklist. 

Essentially ensuring that ntp or something similar is running, and that all nodes have the same UTC datetime.

Skew in clocks can result in a variety of issues, including internode issues.

**Goals**

- [x] Update [the Software Checklist](https://min.io/docs/minio/linux/operations/checklists/software.html) with system clock sync
- [x] Maybe talk about checking ntp and timedatectl ? 
- [x] This could be good to add as a pre-requisite to our deploy tutorials, though it somewhat falls under the "Make sure your nodes aren't broken before installing MinIO" kind of rule

### PR DESCRIPTION
**Pull request by djwfyi**
Adds a checklist item to software about configuring time synchronization. Adds a prereq to the multi-node deployment about time sync.

Closes #778

### CODE REVIEW

File: source/operations/install-deploy-manage/deploy-minio-multi-node-multi-drive.rst [hunk 1]
```diff
@@ -103,6 +103,15 @@ Local JBOD Storage with Sequential Mounts
...
    NFSv4 for best results.
 
+Time Synchronization
+~~~~~~~~~~~~~~~~~~~~
+
+Multi-node systems must maintain synchronized time and date, as reads and writes happen across nodes.
+Make sure all nodes sync to the same time server regularly.
+Operating systems vary for methods used to synchronize time and date, such as with ``ntp``, ``timedatectl``, or ``timesyncd``.
+
+Check the documentation for your operating system for how to set up and maintain accurate time syncs across your nodes.
```
Comment (feorlen) [inline source/operations/install-deploy-manage/deploy-minio-multi-node-multi-drive.rst:113]: Was this intended to be `syncs`? I'm not sure.

**Review by feorlen — APPROVED** [review]
LGTM mod a question that may be me not knowing the domain 👍🏻

**Review by harshavardhana — APPROVED** [review]

File: source/operations/install-deploy-manage/deploy-minio-multi-node-multi-drive.rst [hunk 2]
```diff
@@ -103,6 +103,15 @@ Local JBOD Storage with Sequential Mounts
    For deployments that *require* using network-attached storage, use
    NFSv4 for best results.
 
+Time Synchronization
+~~~~~~~~~~~~~~~~~~~~
+
+Multi-node systems must maintain synchronized time and date, as reads and writes happen across nodes.
```
Comment (ravindk89) [inline source/operations/install-deploy-manage/deploy-minio-multi-node-multi-drive.rst:109]: I would make this a little more generic

```suggestion
Multi-node systems must maintain synchronized time and date to maintain stable internode operations and interactions."
```

IIRC there are non-read/write ops that could be affected by clock skew, such as TLS connections.

File: source/operations/install-deploy-manage/deploy-minio-multi-node-multi-drive.rst (hunk 1 above)
Comment (djwfyi) [reply source/operations/install-deploy-manage/deploy-minio-multi-node-multi-drive.rst:113]: I like that suggestion.

File: source/operations/install-deploy-manage/deploy-minio-multi-node-multi-drive.rst (hunk 2 above)
Comment (djwfyi) [reply source/operations/install-deploy-manage/deploy-minio-multi-node-multi-drive.rst:109]: Done.

**Pull request merged**